.github/workflows/test-ios-simulator.yaml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v3
 
+      - name: Switch Xcode version
+        run: sudo xcode-select --switch /Applications/Xcode_14.0.1.app/Contents/Developer
+
```

Comment:
Does Bluepill not support latest Xcode? If yes, then I'm against using it - it'll be yet another thing that can cause builds to fail because of version mistmatch.

Reply:
Found and option `unsafe-skip-xcode-version-check` and luckily it works with latest version